tfx/examples/cifar10/README.md

```diff
@@ -0,0 +1,88 @@
+
+# CIFAR-10 Transfer Learning and MLKit integration Example
+This example illustrates how to use Transfer Learning for image classification with TFX, and use trained model to do object detection with [MLKit](https://developers.google.com/ml-kit)
+
+## Instruction
+
+Create a Python 3 virtual environment for this example and activate the
+`virtualenv`:
+
+```
+virtualenv -p python3.7 cifar10
+source ./cifar10/bin/activate
+```
+
+Next, install the dependencies required by the CIFAR-10 example (appropriate
+version of TF2 will be installed automatically).
+
+```
+pip install tfx
+pip install tflite-support
+pip install flatbuffers
+```
+
+Then, clone the tfx repo and copy cifar10/ folder to home directory:
+
+```
+git clone https://github.com/tensorflow/tfx ~/tfx-source && pushd ~/tfx-source
+cp -r ~/tfx-source/tfx/examples/cifar10 ~/
+```
+### Dataset
+There is a subset of CIFAR10 (100 images) available in the data folder. To prepare the whole dataset, first create a script and run the following Python code:
+```
+import tensorflow_datasets as tfds
+ds = tfds.load('cifar10', data_dir='./cifar10/data/',split=['train', 'test'])
+```
+Then, create sub-folders for different dataset splits and move different splits to corresponding folders.
+```
+cd cifar10/data
+mkdir train_whole
+mkdir test_whole
+mv cifar10/3.0.2/cifar10-train.tfrecord-00000-of-00001 train_whole
+mv cifar10/3.0.2/cifar10-test.tfrecord-00000-of-00001 test_whole
+```
+You'll find the final dataset under `train_whole` and `test_whole` folders.
+Finally, clean up the data folder.
+```
+rm -r cifar10
+```
+### Train the model
+Execute the pipeline python file :
+```
+python ~/cifar10/cifar_pipeline_native_keras.py
+```
+The trained model is located at `~/cifar10/serving_model_lite/tflite`
+
+### Add MetaData to the trained model
```

Comment:
This is no longer needed right? because you moved the metadata management to within the pipeline (which is great by the way).